src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/ControllerAdvice.kt

```diff
@@ -17,7 +21,17 @@ import uk.gov.justice.digital.hmpps.whereabouts.services.InvalidCourtLocation
 import uk.gov.justice.digital.hmpps.whereabouts.services.ValidationException
 import javax.persistence.EntityNotFoundException
 
-@RestControllerAdvice(basePackageClasses = [AttendanceController::class, AttendanceStatisticsController::class, AttendancesController::class, CourtController::class, AgencyController::class, LocationController::class, CellMoveController::class])
+@RestControllerAdvice(
+  basePackageClasses = [
+    AttendanceController::class,
```

Comment:
Defining the classes here seems redundant as the controller advice functionality seems to work regardless. I'll remove them in the next PR